airbyte-integrations/connector-templates/source-configuration-based/README.md.hbs

```diff
@@ -1,115 +1,91 @@
-This module should contain a `pre_connector_install` and `post_connector_install` async function that will mutate the base image and the connector container respectively.
-It will be imported at runtime by our build process and the functions will be called if they exist.
-
-Here is an example of a `build_customization.py` module:
-```python
-from __future__ import annotations
 
-from typing import TYPE_CHECKING
-
-if TYPE_CHECKING:
-    # Feel free to check the dagger documentation for more information on the Container object and its methods.
-    # https://dagger-io.readthedocs.io/en/sdk-python-v0.6.4/
-    from dagger import Container
 
+### Create credentials
+**If you are a community contributor**, follow the instructions in the [documentation](https://docs.airbyte.com/integrations/sources/{{dashCase name}})
+to generate the necessary credentials. Then create a file `secrets/config.json` conforming to the `src/source_{{snakeCase name}}/spec.yaml` file.
+Note that any directory named `secrets` is gitignored across the entire Airbyte repo, so there is no danger of accidentally checking in sensitive information.
+See `sample_files/sample_config.json` for a sample config file.
 
-async def pre_connector_install(base_image_container: Container) -> Container:
-    return await base_image_container.with_env_variable("MY_PRE_BUILD_ENV_VAR", "my_pre_build_env_var_value")
 
-async def post_connector_install(connector_container: Container) -> Container:
-    return await connector_container.with_env_variable("MY_POST_BUILD_ENV_VAR", "my_post_build_env_var_value")
+### Locally running the connector
+```
+poetry run source-{{dashCase name}} spec
+poetry run source-{{dashCase name}} check --config secrets/config.json
+poetry run source-{{dashCase name}} discover --config secrets/config.json
+poetry run source-{{dashCase name}} read --config secrets/config.json --catalog sample_files/configured_catalog.json
 ```
 
-#### Build your own connector image
-This connector is built using our dynamic built process in `airbyte-ci`.
-The base image used to build it is defined within the metadata.yaml file under the `connectorBuildOptions`.
-The build logic is defined using [Dagger](https://dagger.io/) [here](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/pipelines/builds/python_connectors.py).
-It does not rely on a Dockerfile.
-
-If you would like to patch our connector and build your own a simple approach would be to:
-
-1. Create your own Dockerfile based on the latest version of the connector image.
-```Dockerfile
-FROM airbyte/source-{{dashCase name}}:latest
-
-COPY . ./airbyte/integration_code
-RUN pip install ./airbyte/integration_code
-
-# The entrypoint and default env vars are already set in the base image
-# ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
-# ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
+### Running tests
+To run tests locally, from the connector directory run:
```

Comment:
Nit: I love leaving empty an line between markup elements like headings and code blocks to make sure all md renderers are happy. But, if it works, I'm okay.